File: atest.py
import sys




if __name__ == '__main__':
  flag =     1
  write = ""
  while    flag ==  1:
    print(       "Write Something:")
    write = input("") if sys.version_info >= (3,0) else raw_input("")
    print(       "You typed:" + write)
    if    write ==     "q":
      flag=0


  print(       "Quit Program")

File: atest.go
package main

import "fmt"


func main() {
     flag := 1;
var input string
   for flag   ==    1 {
   fmt.Println(  "Write Something:")
     fmt.Scanln(&input)
     fmt.Printf( "You typed: %s\n", input)
     if input      ==      "q" {
      flag=0
     }
   }
   fmt.Println("Quit Program")
}
